assets/js/src/core/components/tree/components/context-menu/context-menu.tsx

```diff
@@ -71,7 +70,7 @@ export const TreeContextMenu = ({ children, node }: TreeContextMenuProps): React
 
           if (formData.has('file')) {
             // @ts-expect-error - marvin will know why
-            await dispatch(assetUpload.initiate({ body: formData, parentId: parseInt(node.id) })).unwrap()
+            await dispatch(assetUpload.initiate({ body: formData, parentId: parseInt(node.id) }))
```

Comment:
We should move the asset related code out of the tree component, because it's not directly related to UI components.

In general, there should be no more business logic be added under the components folder,
instead we should locate this specific tree logic somewhere close to modules/asset/tree.

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/527/commits/7ac29efdc39882e0150aa4f8cc710aaa6b66fc4c